web/packages/teleport/src/Assist/Conversation/AccessRequest.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { ButtonPrimary } from 'design';
+import Flex from 'design/Flex';
+
+import { Resources } from 'teleport/Assist/Conversation/AccessRequests/Resources';
+
+import type { Resource } from 'teleport/Assist/types';
+
+interface AccessRequestProps {
+  resources: Resource[];
+  reason: string;
+}
+
+const Container = styled.div`
+  padding: 15px 15px 15px 17px;
```

Comment:
Consider using consistent spacing values from the theme instead of hardcoding seemingly arbitrary ones. This goes for every padding, margin and other spacing value in this PR.

https://github.com/gravitational/teleport/blob/61163ce5078d5e0231dee1f6ef7b09ec15488ae2/web/packages/teleterm/src/ui/TopBar/AdditionalActions.tsx#L255

This will ensure that Assist's UI doesn't look out of place when embedded within the Web UI.

These two articles have good examples at the beginning of what happens when spacing is not consistent:

* https://marvelapp.com/blog/harmonious-spacing-system-faster-design-dev-handoff/
* https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62